mkosi/resources/mkosi.md

```diff
@@ -1832,6 +1839,14 @@ are mounted into the current working directory before running the script
 in the current working directory. `$SRCDIR` is set to point to the
 current working directory. The following scripts are supported:
 
+* If **`mkosi.sync`** (`SyncScripts=`) exists, it is executed before the
+  image is built. This script may be used to update various sources that
+  are used to build the image. One use case it to run `git pull` on
+  various source repositories before building the image. Specifically,
+  the `BuildSourcesEphemeral=` setting does not apply to sync scripts,
+  and so sync scripts can be used to update build sources even if
```

Comment:
```suggestion
  so sync scripts can be used to update build sources even if
```
or
```suggestion
  and sync scripts can be used to update build sources even if
```
or
```suggestion
  so that sync scripts can be used to update build sources even if
```